contrib/learn_resource_weights.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+import json
+import logging
+
+logging.basicConfig(format='[%(levelname)s] %(message)s')
+
+
+def parse_commandline():
+    """Parse commandline arguments"""
+    import argparse
+    parser = argparse.ArgumentParser(description='export and analyze resources from statistics',
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('command', choices=[
+                        'parse', 'analyze'], help='path of benchmark results')
```

Comment:
This seems to copy the help text from the option `basedir`. How about:

```suggestion
                        'parse', 'analyze'], help='task to perform')
```